lib/pyfrc/sim/ui.py

```diff
@@ -200,25 +200,25 @@ def _setup_widgets(self, frame):
         slot.pack(side=tk.TOP, fill=tk.BOTH, padx=5)
         
         # CAN
-#       self.can_slot = tk.LabelFrame(csfm, text='CAN')
-#       self.can_slot.pack(side=tk.LEFT, fill=tk.BOTH, expand=1, padx=5)
-#       self.can_mode_map = {}
-#                         tsrxc.kMode_CurrentCloseLoop: 'PercentVbus',
-#                         tsrxc.kMode_DutyCycle:'PercentVbus',
-#                         tsrxc.kMode_NoDrive:'Disabled',
-#                         tsrxc.kMode_PositionCloseLoop:'Position',
-#                         tsrxc.kMode_SlaveFollower:'Follower',
-#                         tsrxc.kMode_VelocityCloseLoop:'Speed',
-#                         tsrxc.kMode_VoltCompen:'Voltage'
-#                      }
+        self.can_slot = tk.LabelFrame(csfm, text='CAN')
+        self.can_slot.pack(side=tk.LEFT, fill=tk.BOTH, expand=1, padx=5)
+        self.can_mode_map = {
+            tsrxc.kMode_CurrentCloseLoop: 'PercentVbus',
```

Comment:
do you know why the old sim used internal instead of `wpilib.CANTalon.ControlMode.*`?